.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+**Please check or ~strike-through~ all of the following**
+This pull-request
+- [ ] resolves #<issue-id>
+- [ ] has a meaningful title
+- [ ] complies to the [code of conduct](../CODE_OF_CONDUCT.md)
```

Comment:
Clicking on "code of conduct" link brings me to a 404 page. I think, it has to be `../blob/master/CODE_OF_CONDUCT.md`

Reply:
True, will change it!